src/Sarif.Sarifer/GenerateTestDataCommand.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel.Design;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Sarif.Viewer.Interop;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    internal class GenerateTestDataCommand : MenuCommand
+    {
+        private static SarifViewerInterop s_viewerInterop;
```

Comment:
>static [](start = 16, length = 6)

Does this really need to be static?

Reply:
I didn't know how to get an instance method passed in as the "event handler" in the call to the base class ctor on Line 21. Would love to change that.

---
In reply to: [514612188](https://github.com/microsoft/sarif-visualstudio-extension/pull/234#discussion_r514612188) [](ancestors = 514612188)

Reply:
Fixed, using your suggestion to aggregate rather than inherit from `MenuCommand`.

---
In reply to: [514613555](https://github.com/microsoft/sarif-visualstudio-extension/pull/234#discussion_r514613555) [](ancestors = 514613555,514612188)